client/src/components/ChartBars.vue

```diff
@@ -12,29 +12,47 @@ export default {
     rounds: {
       type: Array,
       required: true
+    },
+    tab: {
+      type: String,
+      required: true
     }
   },
+  computed: {
+    chartdata () {
+      return {
+          labels: this.team.coop ? ["Victories", "Defeats"] : this.getPlayers(),
+          datasets: [
+            {
+              label: "Victories",
+              backgroundColor: "#ec8506",
+              data: this.team.coop ? this.getCoopStat() : this.getPlayersStat()
+            }
+          ]
+        }
+    },
+  },
   watch: {
-    rounds(value) {
-      value ? this.setChart() : this._chart.destroy();
+    rounds() {
+      this.updateChart()
+    },
+    tab(val) {
+      if (val === 'tab-1') this.updateChart()
     }
   },
   mounted() {
     this.setChart();
   },
   methods: {
+    updateChart () {
+      if (this.$data._chart) {
+        this.$data._chart.data = this.chartdata;
+        this.$data._chart.update();
+      }
```

Comment:
you don't need $data. You can go for this._chart directly

Reply:
This does not work for me without the `$data` (and I'm certain it should not). `this._chart` is `undefined`. 
Docs: https://vue-chartjs.org/api/#chart-js-object
Source: https://github.com/apertureless/vue-chartjs/blob/develop/src/BaseCharts.js

Reply:
fair enough:)